src/rabbit_reader.erl

```diff
@@ -767,6 +767,7 @@ wait_for_channel_termination(N, TimerRef,
                                "error while terminating:~n~p~n",
                         [self(), ConnName, VHost, User#user.username,
                          CS, Channel, Reason]),
+                    rabbit_core_metrics:channel_closed(ChPid),
```

Comment:
Should a `rabbit_event` event be emitted here, since `terminate/2` for channel processes can't be relied on in the case of `brutal_kill` (or so it seems)? Like here: https://github.com/rabbitmq/rabbitmq-common/blob/b83f586974292cecf884c0369bbdcd3e2ed5edee/src/rabbit_channel.erl#L647.